google-cloud-clients/google-cloud-bigquerydatatransfer/README.md

```diff
@@ -115,4 +115,4 @@ Apache 2.0 - See [LICENSE] for more information.
 
 [cloud-bigquerydatatransfer]: https://cloud.google.com/bigquery/docs/transfer-service-overview
 [bigquerydatatransfer-product-docs]: https://cloud.google.com/bigquery/docs/transfer-service-overview
-[bigquerydatatransfer-client-lib-docs]: https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/index.html?com/google/cloud/bigquerydatatransfer/v1/package-summary.html
+[bigquerydatatransfer-client-lib-docs]: https://googlecloudplatform.github.io/google-cloud-java/google-cloud-clients/apidocs/index.html?com/google/cloud/bigquerydatatransfer/v1/package-summary.html
```

Comment:
This link is giving me a 404 in the frame (and some others too, but not all of them) - it isn't just the new link, the old link seems to have trouble too.

Reply:
The link was broken before, as well as compute link. This is not related to the change. I updated the two broken links to point to proper location. Pleaes let me know if you found other broken links (except example links, they are now broken, it is a "known issue").